IRremoteESP8266.cpp

```diff
@@ -800,6 +797,13 @@ bool IRrecv::decode(decode_results *results) {
     return false;
   }
 
+  for (int i = 0; i < irparams.rawlen; i++) {
+    if (irparams.rawbuf[i] < 0)
```

Comment:
This will never be true, as rawbuf[i] is an unsigned int. See: https://github.com/markszabo/IRremoteESP8266/blob/master/IRremoteInt.h#L208

Probably safe to determine if the timing is sufficiently huge, that we've wrapped.
i.e. If we have a timing of, say > approx 50,000usecs/50ms (0xCFFF = 53247) we may have wrapped.